common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -440,7 +441,8 @@ public EncryptionType getEncryptionType(@NonNull final String data) throws Unsup
 
         if (encryptionType == EncryptionType.USER_DEFINED) {
             if (isBrokerProcess()) {
-                if (COMPANY_PORTAL_APP_PACKAGE_NAME.equalsIgnoreCase(packageName)) {
+                if (COMPANY_PORTAL_APP_PACKAGE_NAME.equalsIgnoreCase(packageName) ||
+                        BROKER_HOST_APP_PACKAGE_NAME.equalsIgnoreCase(packageName)) {
```

Comment:
should we have a static utility method for this?

Reply:
we could but I would consider it an overkill....besides I don't know what the name that method